tests/test_widgets.py

```diff
@@ -8,22 +9,39 @@
 )
 
 
+def example_number(region_code: str) -> PhoneNumber:
+    number = phonenumbers.example_number(region_code=region_code)
+    e164 = phonenumbers.format_number(
+        numobj=number, num_format=phonenumbers.PhoneNumberFormat.E164
+    )
+    return PhoneNumber.from_string(phone_number=e164, region=region_code)
+
+
+EXAMPLE_NUMBERS = (
+    example_number(region_code="US"),
+    example_number(region_code="CA"),
+    example_number(region_code="GB"),
+    example_number(region_code="GG"),
+    example_number(region_code="PL"),
```

Comment:
Can also use an italian phone number with a significant leading 0 to cover the missed case `+390612345678`.

Reply:
This is a really good idea. Do you mind using `+390212345678` instead because this is an example number for Italy? It would fit well to the previous test cases.

```pycon
>>> import phonenumbers
>>> it = phonenumbers.example_number('IT')
>>> phonenumbers.format_number(it, phonenumbers.PhoneNumberFormat.E164)
'+390212345678'
```

Reply:
Do you know if all Italian numbers have a significant leading zero? If so, then I’m fine using an example number. Otherwise, I would prefer to keep the input static so that we know the leading-zero case is covered, even if `phonenumbers` changes their example number for the italian region.

Reply:
According to [ITU-T](https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200006B0001PDFE.pdf) all Italian geographic numbers starts with zero and `phonenumbers.example_number` returns a valid fixed-line number for the specified region. So this is confirmed.

Reply:
This is covered with 2a9a1d9.

Reply:
I may have missed something, but the nomadic communications numbers (I’m guessing mobile) start with 55, not 0.

Constructing the phone number ourselves is preferable, because we do not control what example number is chosen when only specifying the "IT" region. WDYT?

Reply:
Basically `example_number` is just a w wrapper over `example_number_for_type` to always return `PhoneNumberType.FIXED_LINE`. Those numbers always start with 0 in Italy.

```pycon
>>> from phonenumbers import PhoneNumberFormat, PhoneNumberType, example_number_for_type, format_number
>>> 
>>> example = example_number_for_type('IT', PhoneNumberType.FIXED_LINE)
>>> print(format_number(example, PhoneNumberFormat.E164))
+390212345678
```

What you mentioned is `PhoneNumberType.VOIP` and it starts with 55 indeed.

```pycon
>>> from phonenumbers import PhoneNumberFormat, PhoneNumberType, example_number_for_type, format_number
>>> 
>>> example = example_number_for_type('IT', PhoneNumberType.VOIP)
>>> print(format_number(example, PhoneNumberFormat.E164))
+395512345678
```
Mobile numbers on the other hand are type `PhoneNumberType.MOBILE` and starts with 3.

```pycon
>>> from phonenumbers import PhoneNumberFormat, PhoneNumberType, example_number_for_type, format_number
>>> 
>>> example = example_number_for_type('IT', PhoneNumberType.MOBILE)
>>> print(format_number(example, PhoneNumberFormat.E164))
+393123456789
```

As long as Autorità per le Garanzie nelle Comunicazion keeps this format there should be no changes and there always will be leading zero. If they don't our test case would also fail because our static example number would no longer be correct.

I understand your concern and I think this is safe to transfer this responsibility to our dependency because fixed line numbers in Italy share this feature and we always get correct example number.

Reply:
Ok. The default being `FIXED_LINE` works for me.